packages/permission-controller/src/PermissionController.test.ts

```diff
@@ -1850,7 +1851,7 @@ describe('PermissionController', () => {
       expect(() =>
         controller.removeCaveat(
           origin,
-          PermissionNames.wallet_noopWithRequiredCaveat,
+          PermissionNames.wallet_noopWithRequiredCaveat as any,
```

Comment:
Why the type cast?

Reply:
It works on my end if I just remove the type cast. `yarn build` completes without errors.

Reply:
It seems to work if you change the method implementation args to `RestrictedMethodOptions<Json[]>`.

🤔

Reply:
I see. In that case I would prefer `// @ts-expect-error`, so we can catch cases where this value becomes valid?

Reply:
It's not 😄